### PR TITLE
+bigstringaf.0.5.1

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/bigstringaf/bigstringaf.0.5.1/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/bigstringaf"
+bug-reports: "https://github.com/inhabitedtype/bigstringaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/bigstringaf.git"
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/bigstringaf/bigstringaf.0.5.1/opam]:
```suggestion
```
You don't seem to use any dune substitutions

File: packages/bigstringaf/bigstringaf.0.5.1/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/bigstringaf/bigstringaf.0.5.1/opam]:
```suggestion
  "dune" {build & >= "1.6"}
```
According to your `dune-package` file, this package requires dune >= 1.6

File: packages/bigstringaf/bigstringaf.0.5.1/opam (hunk 1 above)
Comment (seliopou) [reply packages/bigstringaf/bigstringaf.0.5.1/opam]: Is there any harm in leaving these around?

File: packages/bigstringaf/bigstringaf.0.5.1/opam (hunk 2 above)
Comment (seliopou) [reply packages/bigstringaf/bigstringaf.0.5.1/opam]: Fixed in inhabitedtype/bigstringaf#38. I don't know if dune does validation, but the build files certainly don't use any new features.

File: packages/bigstringaf/bigstringaf.0.5.1/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/bigstringaf/bigstringaf.0.5.1/opam]: no but it's just some more noise in the package description that could be avoided

File: packages/bigstringaf/bigstringaf.0.5.1/opam (hunk 2 above)
Comment (kit-ty-kate) [reply packages/bigstringaf/bigstringaf.0.5.1/opam]: ok thanks! This PR will need to be updated though

**Pull request closed without merging**